sinks/factory.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sinks
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strconv"
+
+	"github.com/GoogleCloudPlatform/heapster/extpoints"
+	sink_api "github.com/GoogleCloudPlatform/heapster/sinks/api"
+	"github.com/GoogleCloudPlatform/heapster/sinks/gcl"
+	"github.com/GoogleCloudPlatform/heapster/sinks/gcm"
+	"github.com/GoogleCloudPlatform/heapster/sinks/influxdb"
+	"github.com/golang/glog"
+)
+
+func init() {
+	extpoints.SinkFactories.Register(CreateInfluxdbSink, "influxdb")
```

Comment:
Could these registrations be moved into their respective packages to keep consistent registration pattern?